src/helm/benchmark/run_specs/melt_run_specs.py

```diff
@@ -291,3 +292,142 @@ def get_math_spec(
         + get_generative_harms_metric_specs(),
         groups=["melt"] + groups,
     )
+
+
+@run_spec_function("melt_text_classification_vsmec")
+def get_melt_text_classification_vsmec_spec(max_train_instances: int = 0) -> RunSpec:
```

Comment:
Remove `max_train_instances` as an argument from all run spec functions. This is already a [run expander](https://crfm-helm.readthedocs.io/en/latest/run_entries/#run-expanders), so `melt_text_classification_vsmec:model=text,max_train_instances=5` will work without this argument.

i.e. just do

```python3
    adapter_spec = get_generation_adapter_spec(
        instructions=instruction,
        input_noun="Bình luận",
        output_noun="Loại cảm xúc",
        max_tokens=50,
        max_train_instances=5,
        multi_label=False,
    )
```